src/reducers/app/reducers.ts

```diff
@@ -1,16 +1,54 @@
+import { Token, UserInfo, AppState, UserState } from "../../types";
+import { Actions, AccessTokenTypeKeys, UserInfoTypeKeys } from "./types";
+import { ActionTypeStates } from '../../models'
 
-
-
-export interface AppState {
-
-}
-
-let initialState: AppState = {
+let initialTokenState: Token = {
+  accessToken: null,
 };
+/**
+ * token
+ * @param state
+ * @param action
+ */
+export const token = (
+  state: Token = initialTokenState,
+  action: Actions
+): Token => {
+  switch (action.type) {
+    case AccessTokenTypeKeys.SET_ACCESS_TOKEN:
+      state = { ...state, ...action.payload };
```

Comment:
based on doc reducers should take the current state and an action as arguments and return a new state result

Reply:
![image](https://user-images.githubusercontent.com/76498759/110969768-4d0aba80-8316-11eb-8c7c-d534961b24fa.png)
Yes, it does

Reply:
ok, it should work fine